### ISSUE TITLE
[question] Create Azure DevOps task

### ISSUE THREAD

**Issue by vanmash** [aquasecurity/tfsec#1148]
Hi

please consider creating Azure DevOps task

**Comment by shubhamsre** [comment]
https://github.com/ritesh-modi/terraform-static-code-tfsec-task

I was about to start working on it but looks like it is in progress and already there.

**Comment by cliftonz1** [comment]
I agree with @vanmash. We are looking to adapt this now with our Azure CI/CD pipeline because of Thoughtworks.

**Comment by shubhamsre** [comment]
@cliftonz1 I am no longer working on it, I developed a task but sadly it was still on my local when I left the other org xD. Yes this can be raised in azure tasks official repository, but still it's aquasec's product so they must be having a plan for it too.

**Comment by cliftonz1** [comment]
@shubhamsre Thank you for the response.
@liamg  Do you know if this is currently on your roadmap?

**Comment by liamg** [comment]
Hey @cliftonz1 yes, this is coming soon!

**Comment by liamg** [comment]
I've got something running locally but thanks to the Azure DevOps partial outage today and the fact requesting credits from MS takes 2-3 business days, it may be a while before I can test it.

Rest assured though, it's on the way!

**Comment by liamg** [comment]
Here you go, sorry about the delay:

https://marketplace.visualstudio.com/items?itemName=AquaSecurityOfficial.tfsec-official

Please raise any issues/feature requests on the [task repository](https://github.com/aquasecurity/tfsec-azure-pipelines-task).

**Comment by cliftonz1** [comment]
Thank you very much!
